File: กดวงกลมตามจังหวะเพลงได้แม่นขนาดไหนกันเชียว.py
n300 = float(input())
n100 = float(input())
n50 = float(input())
n0 = float(input())
Accuracy = ((50 * n50) + (100 * n100) + (300 * n300)) / (300 * (n0 + n50 + n100 + n300))
percent = Accuracy*100
percent2 = "{:.2f}".format(percent)
print('{}%'.format(percent2))

File: Dev_Lab Unofficial/C++/กดวงกลมตามจังหวะเพลงได้แม่นขนาดไหนกันเชียว.cpp
#include <iostream>
#include <iomanip>
using namespace std;


int main()
{
	float n300,n100,n50,n0;
  cin >> n300 >> n100 >> n50 >> n0;
  float Accuracy = ((50 * n50) + (100 * n100) + (300 * n300)) / (300 * (n0 + n50 + n100 + n300));
  cout << fixed << setprecision(2);
  cout << Accuracy*100 << "%";
	return 0;
}